src/services/regCheckDB.js

```diff
@@ -0,0 +1,33 @@
+const users = require('../dummy/userDB');
+
+const CheckUserName = (input) => {
+  const user = /^[A-Za-z0-9]\w{0,}$/;
+  if (input && user.test(input)) {
+    return true;
+  }
+  return false;
+};
+
+const CheckPassword = (input) => {
+  const passw = /^[A-Za-z0-9]\w{7,}$/;
+  if (input && passw.test(input)) {
+    return true;
+  }
+  return false;
+};
+
+const containsUser = (obj) => {
+  const userNames = users.getNames(users.userDB);
+  for (let i = 0; i < userNames.length; i += 1) {
+    if (userNames[i] === obj.username) {
+      return true;
+    }
+  }
+  return false;
+};
+
+module.exports = {
+  CheckUserName,
+  CheckPassword,
+  containsUser,
```

Comment:
inconsistent function names, use camel case

Reply:
i changed it